README.md

```diff
@@ -46,7 +46,9 @@ Book](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/inde
 - Plaintext data is expected/shown as a UTF-8 string (input data of `sign`, output data of
    `decrypt`).
 - Ciphertext data is expected/shown as base 64 (output data of `sign`, input data of `decrypt`).
-- Exported public keys are formatted in PEM.
+- Exported public keys are formatted in PEM. RSA keys are encoded in [`RSAPublicKey`](https://tools.ietf.org/html/rfc3279.html#section-2.3.1)
```

Comment:
Note that I wrote above:

> Unless specified otherwise below, the data format expected by the commands is the same as describe in the [Parsec
Book](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/index.html).

and that this info is on the [`PsaExportPublicKey`](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_export_public_key.html) page for example.

But that said nothing wrong about writing it here as well. I just did that so we wouldn't have to write the format for every key type 😋

Reply:
Less indirection is better tbh

Reply:
And also because we're not really transparent about default key types and all of that. Like, you can find out that the key is P-256 using `list-keys` (or by checking the rustdocs or reading the code) and that we use `psa-export-public-key` (since it's the only operation we can use), but it's not very explicit